package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "_s",
+  "version": "1.0.0",
+  "license": "GPL-2.0+",
+  "scripts": {
+    "build:css": "npm run css-compile && npm run css-prefix && npm run css-format && npm run css-fix",
+    "compile:css": "node-sass sass/style.scss style.css --output-style expanded --indent-type tab --indent-width 1",
+    "prefix:css": "postcss style.css -u autoprefixer -o style.css --no-map",
+    "lint:css": "stylelint style.css",
+    "lint:css": "stylelint 'sass/**/*.scss' --syntax scss --config node_modules/stylelint-config-wordpress/scss.js",
```

Comment:
I think this should be `lint:scss`

Reply:
Yeah the latter one.

Reply:
When I run `npm install` the line `"lint:css": "stylelint style.css",` was removed automatically. Probably because there can't be two same names.

Reply:
@samikeijonen I've updated this. I had also forgotten to update the names when used within the build command. Should be working correctly now.